jujugui/static/gui/src/app/models/models.js

```diff
@@ -244,20 +244,6 @@ YUI.add('juju-models', function(Y) {
       this.set(
           'environmentConfig',
           Y.merge(this.get('environmentConfig'), changeConfig));
-      // If there are any fields where the local version is different from the
-      // new environment fields that means that it has been changed locally.
-      // XXX We will want to enable conflicted field support for the react
-      // stuff in the future but for now it's only causing issues with the UI.
-      if (!window.flags || !window.flags.react) {
-        if (conflicted.length > 0) {
```

Comment:
If you're not using `conflicted` anymore then you can remove how it's generated above.